isaric/parsers/isaric-rapid.toml

```diff
@@ -641,11 +646,19 @@
     { field = "daily_corticost_cmoth" },
   ]
 
+[visit.treatment_cardiovascular_support]
+  combinedType = "any"
+  fields = [
+    { fieldPattern = ".*extracorp_prtrt", ref = "Y/N/NK" },
```

Comment:
like the field pattern method .. 👍

Reply:
Is it ok to combine ECMO with inotropes/vasopressors as cardiovascular support? they seem to be acting by a different mechanism, not sure if it makes more sense from a clinical perspective to keep separate?

Reply:
I see the overarching cardiovascular support variable now. I think this is ok as long as there separate attributes for each of the interventions that are used to create this top-level field? so inotropes_vasopressors, ECMO, CPR, pacing and any others are all indicated separately

Reply:
Yep, we have both cardiovascular_support as a general indicator, similar to oxygen_therapy, with individual attributes for vasopressors, ecmo, pacing etc.